README.md

```diff
@@ -10,6 +10,32 @@ The VSCode api is composed of:
   - If it's an important feature: it requires to use the corresponding service override.
   - If it's some advanced features that don't make a lot of sense on Monaco (scm, tests...), it just throws an error when you try to use it.
 
+## Installation
+
+```bash
+npm install vscode@npm:@codingame/monaco-vscode-api
+npm install -D @types/vscode
+```
+
+⚠️ And add in your package.json ⚠️:
+```json
+{
+  "scripts": {
+    "postinstall": "monaco-treemending",
```

Comment:
Will it be possible to run this script another way and not in `postinstall`? For example, in my Next.js plugin when a user runs `next dev`, should `monaco-treemending` be safe to call each time the server starts using child_process exec?

Reply:
It just applies a patch so you can run it anytime, it will just fail if it was already run

What is the issue with postinstall ?

Reply:
Sounds good! I'd like the install story for end-users of the plugin to be as simple as possible. If anything, I could codemod the package.json when the plugin is first run.